common/src/main/java/org/jboss/pnc/common/json/AbstractModuleGroup.java

```diff
@@ -18,12 +18,7 @@
 package org.jboss.pnc.common.json;
 
 import com.fasterxml.jackson.annotation.JsonTypeInfo;
-import com.fasterxml.jackson.annotation.JsonTypeInfo.As;
 
-@JsonTypeInfo(
-        use = JsonTypeInfo.Id.NAME,
-        include = As.PROPERTY,
-        property = "@module-group",
-        defaultImpl = DefaultModuleGroup.class)
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "@module-group", defaultImpl = DefaultModuleGroup.class)
 public abstract class AbstractModuleGroup {
 }
```

Comment:
Should configure IDEA to add newline at end of file (Editor > General > Ensure an empty line at the end of the file on Save) or add `.editorconfig` to do it automatically.